canonical-websites/www.ubuntu.com
# 266 update arm page

## Done

Move 'Past ARM releases' section to `/download/server/arm`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/arm>
- Check that 'Past ARM releases' section is in page [as per copydoc](https://docs.google.com/document/d/1whVU3DvB9j5k6Ed3UvLZCEN6BU-lZcow4ZefqKQqokg/edit)

## Issue / Card

Fixes [#266](https://github.com/canonical-websites/www.canonical.com/issues/266)
